src/components/fileUpload.js

```diff
@@ -322,12 +498,129 @@
           ],
         },
       },
-      fileList: {
+      listView: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+      },
+      gridView: {
+        display: 'flex',
+      },
+      gridItem: {
+        display: 'flex',
+        borderRadius: '0.3rem',
```

Comment:
Rem values should always end with a `5`. In this case `0,3 * 16 = 4,8`. I don't think you want to render 4,8px? 🤔 Or do I miss something here?

Reply:
You're correct. I miscalculated here. I'll fix the rem values for the rest of the doc. 👍🏻